src/modules/assertions/require_max_file_number.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  requireEvent,
+  requirePr
+} from "#/assertions";
+import { github } from "src/infra/github";
+
+export const  requireMaxFileNumber = async (): Promise <1>  => {
+  const max_files_allowed = 100;
+  requireEvent();
+  const pr = await requirePr();
+  const files = await github.getPullRequestFiles(pr.number);
+  if ((files?.length) > max_files_allowed) {
+    console.log(`Critical error: Number of PR Files > ${max_files_allowed}`);
+    process.exit(1);
+  }
+  return 1;
+};
```

Comment:
This return value isn't used and serves no purpose, so it should be removed.
```suggestion
export const  requireMaxFileNumber = async () => {
  const max_files_allowed = 100;
  requireEvent();
  const pr = await requirePr();
  const files = await github.getPullRequestFiles(pr.number);
  if ((files?.length) > max_files_allowed) {
    console.log(`Critical error: Number of PR Files > ${max_files_allowed}`);
    process.exit(1);
  }
};
```